controllers/tutors.controller.js

```diff
@@ -0,0 +1,31 @@
+const { Tutor } = require('../models/tutor.model');
+
+const getAllTutorsFromDb = async (req, res) => {
+	try {
+		const tutors = await Tutor.find({});
+		res.status(201).json({ response: tutors, success: true });
+	} catch (error) {
+		res.status(500).json({
+			success: false,
+			message: 'Something went wrong',
+			errorMessage: error.message,
+		});
+	}
+};
+
+const createNewTutor = async (req, res) => {
+	try {
+		tutorDetails = req.body;
```

Comment:
I think here variable declaration is missing for tutorDetails

Reply:
Somehow missed it. Thanks for pointing it out. I have updated it.